app/components/Form/ProjectEmissionIntensityReportForm.tsx

```diff
@@ -47,6 +50,21 @@ export const createEmissionIntensityReportUiSchema = (
     );
   };
 
+  const reportDuration = getDurationFromDates(
+    measurementPeriodStartDate,
+    measurementPeriodEndDate
+  );
+  const reportDurationSuffix = reportDuration && (
```

Comment:
Prefix instead of suffix in these names? "Duration" preceding the day count seems like the more meaningful part of the string

Reply:
A good point! Suffix here is referring to the element being a suffix to the "widget content", so it's lining up with the ui property `contentSuffix`